### PR TITLE
Implement use of nameof to remove magic strings

### PR DESCRIPTION
**Pull request by calebnelton**
Implement use of nameof to remove magic strings

### CODE REVIEW

File: Source/Wpf/Prism.Wpf/Extensions/CollectionExtensions.cs [hunk 1]
```diff
@@ -16,11 +16,14 @@ public static class CollectionExtensions
...
-        /// <exception cref="System.ArgumentNullException">An <see cref="System.ArgumentNullException"/> is thrown if <paramref name="collection"/> or <paramref name="items"/> is <see langword="null"/>.</exception>
+        /// <exception cref="ArgumentNullException">An <see cref="System.ArgumentNullException"/> is thrown if <paramref name="collection"/> or <paramref name="items"/> is <see langword="null"/>.</exception>
         public static Collection<T> AddRange<T>(this Collection<T> collection, IEnumerable<T> items)
         {
-            if (collection == null) throw new System.ArgumentNullException("collection");
-            if (items == null) throw new System.ArgumentNullException("items");
+            if (collection == null)
+                throw new ArgumentNullException(nameof(collection));
+            if (items == null)
+
```
Comment (brianlagunas) [inline Source/Wpf/Prism.Wpf/Extensions/CollectionExtensions.cs]: delete this row

**Comment by brianlagunas** [comment]
Looks good, just saw one extra line break after an if statement that should be removed.  Fix that, and I'll merge.

**Comment by calebnelton** [comment]
Whoops, I'm still getting use to submitting pull requests ... didn't mean to sync those last commits. I intend for those to go onto a separate pull request.

**Comment by bartlannoeye** [comment]
Everything committed to the same branch as the PR gets added automatically, so it's best to start a new branch per fix (or wait until after the merge, but that can take some time now and then)

**Pull request merged**